src/interviews/interviews.js

```diff
@@ -19,14 +19,18 @@ export const InterviewsVM = DefineMap.extend('InterviewsVM', {
   },
 
   interviews: {
-    value ({ lastSet, listenTo, resolve }) {
-      this.interviewsPromise.then((interviews) => {
-        resolve(interviews)
-      })
-
-      listenTo(lastSet, (interviews) => {
-        resolve(interviews)
+    value (prop) {
```

Comment:
my above comment might actually be solved with your prop/resolver pattern. i feel like it was likely a canjs warning about child updating parent.

Reply:
yep, this keeps it all happy :)